### PR TITLE
Add the Claudie Intro Video to the readme

### PR DESCRIPTION
**Pull request by MarioUhrik**

### CODE REVIEW

**Comment by Despire** [comment]
the video is aligned to the left, would it possible to center it ? 

Screenshot 2023-02-22 at 12 00 22

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@
   <a href="https://opensource.org/licenses/Apache-2.0"><img src="https://img.shields.io/badge/License-Apache_2.0-blue.svg"></a>
 </p>
 
+## Intro video
```
Comment (wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf) [inline README.md:13]: hi @MarioUhrik, did we think about centering this one, too?

ref
image

**Review by wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf — CHANGES_REQUESTED** [review]

**Comment by wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf** [comment]
I did just comment with the same thing haha @Despire :see_no_evil:

**Comment by MarioUhrik** [comment]
Thanks @wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf @Despire , it's now centered.
I wasn't sure whether to center it, so I didn't at first

**Review by Despire — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -12,7 +12,9 @@
 
 ## Intro video
 
-[![Claudie intro video](https://markdown-videos.deta.dev/youtube/q4xdAiHYxZQ)](https://youtu.be/q4xdAiHYxZQ)
+<p align="center">
+  <a href="https://youtu.be/q4xdAiHYxZQ"><img src="https://markdown-videos.deta.dev/youtube/q4xdAiHYxZQ)"></a>
```
Comment (wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf) [inline README.md]: great!
one thing though, this does not render:
image

there's an extra `)` at the end of `src`... @MarioUhrik

**Review by wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (MarioUhrik) [reply README.md]: fixed

**Review by wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf — APPROVED** [review]
awesome, thanks @MarioUhrik 

image

**Pull request merged**